docs/pages/color_examples.js

```diff
@@ -0,0 +1,388 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Flex, Heading, Link, Text } from 'gestalt';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import Page from '../components/Page.js';
+import A11Y from '../graphics/accessibility.svg';
+import ColorEase from '../graphics/color-examples/colorEase.svg';
+import Consistency from '../graphics/color-examples/consistency.svg';
+
+type PrincipleCardProps = {|
+  color: string,
+  image?: Node,
+  text: string | Node,
+  heading: string,
+|};
+function PrincipleLayout({ color, image, text, heading }: PrincipleCardProps): Node {
+  return (
+    <Flex direction="column" gap={4}>
+      <Box
+        display="flex"
+        alignItems="center"
+        justifyContent="center"
+        height={160}
+        width={260}
+        dangerouslySetInlineStyle={{
+          __style: {
+            backgroundColor: `var(--color-${color})`,
+          },
+        }}
+      >
+        {image}
+      </Box>
+      <Heading size="400">{heading}</Heading>
+      <Box marginBottom={6}>
+        <Text>{text}</Text>
+      </Box>
+    </Flex>
+  );
+}
+
+export default function ColorExamplesPage(): Node {
+  return (
+    <Page title="Color examples">
+      <PageHeader
+        name="Color examples"
+        description={`Color principles and best practices examples of applying color to product UI.
+    `}
+        showSourceLink={false}
+      />
+      <MainSection name="Color principles">
+        <Flex gap={12} alignContent="between" wrap>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={260}>
+            <PrincipleLayout
+              color="teal-spabattical-500"
+              image={<ColorEase />}
+              heading="Ease of content"
+              text={
+                <Text>
+                  Pinterest user content is varied; from rich pinboards featuring every color
+                  imaginable to complex charts and graphs for our business and platform interfaces,
+                  and having a limited range of colors supports consistency, making our product
+                  easier to interact with. Keep in mind that continuous use of high-contrast and
+                  saturated colors can create fatigue and eye strain. See{' '}
+                  <Link inline href="https://gestalt.pinterest.systems/color_usage">
+                    <Text underline>color usage</Text>
+                  </Link>{' '}
+                  for color usage for color tokens guidelines.
+                </Text>
+              }
+            />
+          </Flex.Item>
+          <Flex.Item flex="grow" flexBasis="0%" minWidth={260}>
+            <PrincipleLayout
+              color="orange-firetini-500"
+              image={<Consistency />}
+              heading="Consistency of visuals"
+              text={
+                <Text>
+                  Colors should be applied purposefully as it can convey meaning in numerous ways.
```

Comment:
> For example,[...]

Not sure if the "light and dark mode themes have" are examples of the "Colors should be applied purposefully as it can convey meaning in numerous ways." Maybe we don't need the  "For example,"?

Reply:
I think it's the relation of the purpose concept with "For example, light and dark mode themes have different color hues and saturation for better contrast; however". Not sure of the purpose of the example "For example, light and dark mode themes have different color hues and saturation for better contrast;"